subprojects/gradle/module-types/src/test/kotlin/com/avito/android/ModuleTypesPluginTest.kt

```diff
@@ -48,6 +48,7 @@ class ModuleTypesPluginTest {
         val result = gradlew(
             projectDir, "assemble",
             "-Pavito.moduleTypeValidationEnabled=true",
+            "-PgitBranch=xxx", // todo need for impact plugin
```

Comment:
All additions of gitBranch property happened because ImpactPlugin depends on gitState.
Before that pr gitState can be notDefined and depens on ci=true.
So if we want ImpactPlugin doesn't force git state we can wrap that logic with ci=true.